libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFRootViewManager.m

```diff
@@ -160,14 +174,23 @@ - (void)popViewController:(UIViewController *)viewController
             } else {
                 [strongSelf log:SFLogLevelDebug format:@"popViewController: View controller (%@) is now being dismissed from presentation.", viewController];
                 [[currentViewController presentingViewController] dismissViewControllerAnimated:NO completion:^{
-                    [strongSelf enumerateDelegates:^(id<SFRootViewManagerDelegate> delegate) {
-                        if ([delegate respondsToSelector:@selector(rootViewManager:didPopViewControler:)]) {
-                            [delegate rootViewManager:strongSelf didPopViewControler:viewController];
-                        }
-                    }];
+                      if(strongSelf->_modalViewController) {
+                          [prevController presentViewController:strongSelf->_modalViewController animated:NO completion:^{
+                            strongSelf->_modalViewController = nil;
+                           }];
+                          return;
```

Comment:
(Sorry, previous comment was on the commit, which is a pain in posterity searching of PRs. 🙂)

Won't the `return` statement make it so that the `enumerateDelegates` call doesn't happen?